holidays/holiday_base.py

```diff
@@ -483,6 +483,24 @@ def __getattr__(self, name):
                     + timedelta(days=+int(days) if delta_direction == "past" else -int(days)),
                 )
 
+            # Handle <n> days <past/prior> easter patterns (e.g.,
+            # _add_holiday_8_days_past_easter() or
+            # _add_holiday_5_days_prior_easter()).
+            nth_weekday_of_month_with_delta = re.match(
+                r"_add_holiday_(\d{1,2})_days_(past|prior)_easter",
+                name,
+            )
+            if nth_weekday_of_month_with_delta:
+                (
+                    days,
+                    delta_direction,
+                ) = nth_weekday_of_month_with_delta.groups()
+                return lambda name: self._add_holiday(
+                    name,
+                    self._easter_sunday
```

Comment:
Do we want to explicitly handle cases when the method is invoked for an entity with no `ChristianHolidays` inheritance?

Reply:
Since this is a feature for internal use, let's assume a thoughtful use. Likewise, it's possible to invoke something like `_add_holiday_apr_31()` or `_add_holiday_5_mon_of_feb()` (in non-leap year), or just make a typo and write `_add_holiday_yan_25()`, but we don't check it.